### PR TITLE
pj-43-post_pagination

### PR DESCRIPTION
**Pull request by NykoEugen**

### CODE REVIEW

File: categories/views.py [hunk 1]
```diff
@@ -14,9 +15,12 @@ def category_list(request: HttpRequest) -> HttpResponse:
 def category_detail(request: HttpRequest, category_slug: str) -> HttpResponse:
     category = get_object_or_404(Category, slug=category_slug)
     posts = Post.objects.filter(category_id=category.id, is_active=True)
+    paginator = Paginator(posts, 2)
```
Comment (acman) [inline categories/views.py]: I think 10 posts should be on page

File: categories/views.py [hunk 2]
```diff
@@ -14,9 +15,12 @@ def category_list(request: HttpRequest) -> HttpResponse:
...
 
+    page_number = request.GET.get("page")
+    page_obj = paginator.get_page(page_number)
```
Comment (acman) [inline categories/views.py:21]: There should be some changes in tests

File: templates/posts/post_detail.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {% extends 'content.html' %}
-
+<script type="text/javascript" src="static/js/comment_script.js" defer></script>
```
Comment (acman) [inline templates/posts/post_detail.html]: Don't see any usage of it

**Review by acman — CHANGES_REQUESTED** [review]

File: categories/views.py (hunk 1 above)
Comment (NykoEugen) [reply categories/views.py]: I used 2 for test it

File: categories/views.py (hunk 2 above)
Comment (NykoEugen) [reply categories/views.py:21]: I forgot about test, I'll fix it

File: templates/posts/post_detail.html (hunk 3 above)
Comment (NykoEugen) [reply templates/posts/post_detail.html]: forgot deleted it

**Review by acman — APPROVED** [review]
👍

**Pull request merged**